config/bm_latn.py

```diff
@@ -0,0 +1,28 @@
+"Bambara config with language-specific information."
+
+from pynini import *
+
+GRAPHEMES = union("'", "-",
+                  "A", "B", "C", "D", "E", "F", "G", "H", "I",
```

Comment:
Since we're lowercasing everything, we shouldn't need to include capital letters right?

Reply:
That's true. And the lowercasing and unicode normalization are the very first things that happen, so we shouldn't encounter any capital letters down the road.

Reply:
I removed the capital letters from all the languages with Latin-based orthography.